lib_runtime/mirage_runtime.ml

```diff
@@ -14,6 +14,8 @@
  * OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  *)
```

Comment:
Ah, I hadn't noticed this module. I could have put that annoying helper functor here :-/